MultiplayerMod/Multiplayer/State/MultiplayerGame.cs

```diff
@@ -4,14 +4,24 @@ namespace MultiplayerMod.Multiplayer.State;
 
 public static class MultiplayerGame {
 
+    public static IPlayer Player { get; set; } = null!;
     public static MultiplayerRole Role { get; set; }
-    public static bool WorldSpawned { get; set; }
     public static MultiplayerState State { get; set; } = new();
-    public static MultiplayerObjects Objects { get; set; } = new();
+    public static MultiplayerObjects Objects { get; private set; } = new();
+
+    public static PlayerState CurrentPlayerState {
+        get {
+            if (State.Players.TryGetValue(Player, out var state))
+                return state;
+
+            State.Players.Add(Player, new PlayerState(Player));
+            return State.Players[Player];
```

Comment:
It was supposed to use it like that in place, just `State.Players[client.Player]`.